src/migrations/single_to_cluster.rs

```diff
@@ -62,6 +62,7 @@ pub async fn handle_existing_collections(
                 optimizers_config: Some(collection_state.config.optimizer_config.into()),
                 init_from: None,
                 quantization_config: collection_state.config.quantization_config,
+                strict_mode_config: None,
```

Comment:
why `None`?

Reply:
Good catch! I was adding this a bunch of tests and accidentally also set this to none.